packages/bundle-size/src/commands/uploadReport.js

```diff
@@ -0,0 +1,139 @@
+const { odata, TableClient, TablesSharedKeyCredential, TableTransaction } = require('@azure/data-tables');
+const chalk = require('chalk');
+
+const collectLocalReport = require('../utils/collectLocalReport');
+const { hrToSeconds } = require('../utils/helpers');
+
+/** @typedef {import('../utils/collectLocalReport').BundleSizeReportEntry} BundleSizeReportEntry */
```

Comment:
not needed  
- used only once
- we have already imported that module, so we can use its type metadata directly

```suggestion

```

Reply:
Done in 64cb483.